src/main/java/filemanager/launcher/Launcher.java

```diff
@@ -19,8 +21,11 @@ public static void main(String[] args) throws Exception {
     public void run(FileHandlerConfiguration configuration, Environment environment) {
         Injector guice = Guice.createInjector(new FileServiceBinderModule(configuration));
         ScheduleFileDirectory trackerFileDirectory = guice.getInstance(ScheduleFileDirectory.class);
+        environment.jersey().register(new ClientResource(trackerFileDirectory));
```

Comment:
May I suggest passing ```ClientResource``` instance through Guice instantiation process? So that Guice would be responsible for configuring the instance for us, instead of manual initialization.